src/magic_spell_effect.cpp

```diff
@@ -1894,6 +1894,9 @@ void spell_effect::effect_on_condition( const spell &sp, Creature &caster,
         }
         Creature *victim = creatures.creature_at<Creature>( potential_target );
         dialogue d( victim ? get_talker_for( victim ) : nullptr, get_talker_for( caster ) );
+        const tripoint_abs_ms target_abs = get_map().getglobal( potential_target );
+        write_var_value( var_type::context, "npctalk_var_spell_location", &d,
```

Comment:
*spirit of #77515 is wandering around this string*

Reply:
I'm aware of that, one of us gets a 1 line merge conflict it's not a big deal?

Reply:
nah, i just need to make a reminder to fix it in case it will not make a merge conflict and you just push a variable EoC can't use

Reply:
Oh that's a fair point, I'll draft this until it merges so as not to hold your PR up anyways

Reply:
nah, don't worry, it would be a long time when it will be merged, mergers are particularly slow with such heavy PRs